Content.Server/SyndicateTeleporter/SyndicateTeleporterComponent.cs

```diff
@@ -0,0 +1,27 @@
+using Robust.Shared.Audio;
+
+namespace Content.Server.SyndicateTeleporter;
+
+[RegisterComponent]
+public sealed partial class SyndicateTeleporterComponent : Component
+{
+    [DataField("randomValue")]
+    [ViewVariables(VVAccess.ReadWrite)]
```

Comment:
This is not needed and added automatically (some old files still have it though).
See https://github.com/space-wizards/RobustToolbox/pull/5164